strapi/strapi
# [DEITS] Export schemas

### What does it do?

- Exports schemas (compresses and encrypts)
- created a getSchemas and a streamSchemas methods for local strapi source provider
- created a writable stream for schemas in local destination file provider

### How to test it?

run `yarn strapi export` should have a schemas directory with data (encrypted and compressed)
run `yarn strapi export --encrypt "false" --compress "false"` for data that is not encrypted or compressed
data should match the project schema
